libsel4utils/include/sel4utils/benchmark_track.h

```diff
@@ -45,10 +45,10 @@ seL4_BenchmarkTrackDumpSummary(benchmark_track_kernel_entry_t *logBuffer, size_t
         index++;
     }
 
-    fprintf(fd, "Number of system call invocations %d\n", syscall_entries);
-    fprintf(fd, "Number of interrupt invocations %d\n", interrupt_entries);
-    fprintf(fd, "Number of user-level faults %d\n", userlevelfault_entries);
-    fprintf(fd, "Number of VM faults %d\n", vmfault_entries);
+    fprintf(fd, "Number of system call invocations %lu\n", (unsigned long) syscall_entries);
```

Comment:
We have `SEL4_PRIu_word` for this now

Reply:
I'm happy to update this, but I want to make sure I understand what I'm doing first :). If I traced this correctly, `syscall_entries` is `seL4_Word`, which should already be `unsigned long`. Am I missing something?

I.e. would simply
```c
fprintf(fd, "Number of system call invocations %"SEL4_PRIu_word"\n", syscall_entries);
```
be correct now?

Reply:
yes

Reply:
Actually, they all seem to be `seL4_Word`, so thy all should use `SEL4_PRIu_word`. And using "bla bla %15"SEL4_PRIu_word" bla bla"` will also work. The issue here is that it would be 7 on 32-bit and 15 on 64-bit, but currently we have no constant we can use here. I've ran into this on another place also recently and thinking about a way to resolve this. The current way is always using %15"SEL4_PRIu_word" then. We could add `SEL4_PRIzpu_word` or `SEL4_PRIu_word_zp` to zero-prefix it...